.circleci/config.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+jobs:
+  build-and-test:
+    docker:
+      - image: circleci/node:13
+    working_directory: ~/repo    
+    steps:
+      - checkout      
+      - run: npm install            
+      - run: npm ci
+workflows:
+  version: 2
+  build:
+    jobs:
```

Comment:
you also need a release to npmjs.com script
- Similar but not identical to: https://github.com/SAP/xml-tools/blob/master/.circleci/config.yml#L30-L41

Reply:
I am in the process of building the pipeline now. I am not done 😊